src/services/legacyreports.js

```diff
@@ -0,0 +1,112 @@
+import { Op } from 'sequelize';
+import { userByEmail } from './users';
+import { ActivityReport, ActivityReportCollaborator } from '../models';
+import { logger } from '../logger';
+
+const getLegacyReports = async () => {
+  const reports = await ActivityReport.findAll({
+    where: {
+      legacyId: {
+        [Op.ne]: null,
+      },
+      imported: {
+        [Op.ne]: null,
+      },
+      [Op.or]: [
+        {
+          userId: {
+            [Op.eq]: null,
+          },
+        },
+        {
+          approvingManagerId: {
+            [Op.eq]: null,
+          },
+        },
+        {
+          imported: {
+            otherSpecialists: {
+              [Op.ne]: '',
+            },
+          },
+        },
+      ],
+
+    },
+  });
+  return reports;
+};
+
+export const reconcileApprovingManagers = async (report) => {
+  try {
+    const user = await userByEmail(report.imported.manager);
+    if (user) {
+      await ActivityReport.update({ approvingManagerId: user.id }, { where: { id: report.id } });
```

Comment:
I am not following this comment. Could you clarify?

Reply:
I'm not 100% certain but it seems report is a [modal instance](https://sequelize.org/master/manual/model-instances.html) which would mean `report` has an update method that can be used directly, instead of having to do `ActivityReport.update(.... where: { id: report.id })`. It's pretty minor and the code as is perfectly fine as is.

Reply:
Cool, I tried it and it works, but it requires disabling a eslint warning so I will stick with the current form. That is neat though.

Reply:
Just out of curiosity do you remember what the eslint warning was?

Reply:
no-params-reassign or something like that.